embassy-stm32/src/can/bxcan.rs

```diff
@@ -431,8 +429,13 @@ impl<'c, 'd, T: Instance> CanTx<'c, 'd, T> {
         .await;
     }
 
+    /// Waits for a specific transmit mailbox to become empty
+    pub async fn flush(&self, mb: bxcan::Mailbox) {
+        Self::_flush(mb).await
+    }
+
     /// Waits until any of the transmit mailboxes become empty
-    pub async fn flush_any(&self) {
+    pub async fn _flush_any() {
```

Comment:
`_flush_any`, `_flush_all` should be private.

Also don't prepend with `_`, in Rust it denotes "supress warnings that it's unused", not "it's private" like in Python :)

Reply:
Any suggestions as to what would be a preferred name? `flush_all_impl`?

Reply:
yeah! or `flush_all_inner`.